BaseTools/Source/Python/Trim/Trim.py

```diff
@@ -244,6 +244,21 @@ def TrimPreprocessedVfr(Source, Target):
     except:
         EdkLogger.error("Trim", FILE_OPEN_FAILURE, ExtraData=Target)
 
+## Create a banner for the included ASL file
+#
+# Create a banner to indicate the start and
+# end of the included ASL file
+#

```

Comment:
Can you add a note to this comment about what the final banner will look like (based on sample params)?